src/VirtoCommerce.Platform.Web/Swagger/SwaggerServiceCollectionExtensions.cs

```diff
@@ -83,9 +87,12 @@ public static void AddSwagger(this IServiceCollection services)
 
                 c.DocInclusionPredicate((docName, apiDesc) =>
                 {
-                    if (docName.EqualsInvariant(platformDocName)) return true;
+                    if (docName.EqualsInvariant(platformUIDocName)) return true; // It's an UI endpoint, return all to correctly build swagger UI page
 
-                    var currentAssembly = ((ControllerActionDescriptor)apiDesc.ActionDescriptor).ControllerTypeInfo.Assembly;
+                    if (((ControllerActionDescriptor)apiDesc.ActionDescriptor).ControllerTypeInfo.FullName.StartsWith(platformDocName)) return true; // It's a Platform endpoint. Return platform only
```

Comment:
- add comparing docName
- better use AssemblyName comparing

Reply:
OK. I did it in another way.